README.md

```diff
@@ -4,9 +4,9 @@
 
 ---
 
-[Key Features](#key-features) •
-[Installation](https://openvinotoolkit.github.io/training_extensions/latest/guide/get_started/installation.html) •
-[Documentation](https://openvinotoolkit.github.io/training_extensions/latest/index.html) •
+[Key Features](#key-features)
+[Installation](https://openvinotoolkit.github.io/training_extensions/1.4.2/guide/get_started/installation.html)
+[Documentation](https://openvinotoolkit.github.io/training_extensions/1.4.2/index.html)
```

Comment:
Please don't remove "•".
And would you change the version to 1.4.4 in other PR?

Reply:
I did not removed those at this time ;) I will update them.